src/ray/raylet/scheduling/local_resource_manager_test.cc

```diff
@@ -12,6 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Don't know why macro redefinition happens, but this is failing windows
+// build.
+#pragma warning(disable : 4005)
```

Comment:
which macro is redefined?

Reply:
BOOST_NO_CXX98_FUNCTION_BASE 
see this: https://buildkite.com/ray-project/oss-ci-build-pr/builds/32960#018a1a16-ce52-4591-a9ab-29301082b57b

Reply:
actually - i think the test failure didn't seem to be due to redefinition. https://buildkite.com/ray-project/oss-ci-build-pr/builds/32995#018a1a65-b371-482e-8a62-9fa426c4922b

still fails even though there's no more redefinition on the `local_resource_manager_test`